apps/src/templates/instructions/ContainedLevelResetButton.jsx

```diff
@@ -35,7 +30,7 @@ export const UnconnectedContainedLevelResetButton = ({
     });
   };
 
-  if (userRoleInCourse !== CourseRoles.Instructor || !isEnabled) {
+  if (userRoleInCourse !== CourseRoles.Instructor) {
```

Comment:
Just for my own understanding- was the instructor tab logic loophole here? It's always so hard for me to think through or statements with negation😅

Reply:
The issue was in `TopInstructions` so I fixed it there ([commit](https://github.com/code-dot-org/code-dot-org/pull/48520/commits/4976b8ba7762c5482ee42f2d5ad005dc76b95b6f)).

That's a good point about statements with negation being less clear. Maybe in a follow up I can switch this branch around 🤔 Something like `if (userRoleInCourse === CourseRoles.Instructor) { return <Component> } else { return null; }`